app/src/main/kotlin/cz/covid19cz/erouska/ui/help/data/FaqCategory.kt

```diff
@@ -17,8 +19,14 @@ data class FaqCategory(
 
 fun String?.toFaqCategories(): List<FaqCategory> {
     val categoryType: Type = object : TypeToken<ArrayList<FaqCategory>>() {}.type
-    val structuredQs: ArrayList<FaqCategory>? =
-        Gson().fromJson(this, categoryType)
-    return structuredQs.orEmpty()
+    return try {
+        val structuredQs: ArrayList<FaqCategory>? =
+            Gson().fromJson(this, categoryType)
+        structuredQs.orEmpty()
+    } catch (ex: JsonSyntaxException) {
+        Collections.emptyList()
+    } catch (ex1: JsonParseException) {
```

Comment:
`JsonSyntaxException extends JsonParseException` so 
`catch (ex1: JsonParseException) {..}` should be enough